CONTRIBUTING.md

```diff
@@ -23,3 +23,52 @@ use GitHub pull requests for this purpose. Consult [GitHub Help] for more
 information on using pull requests.
 
 [GitHub Help]: https://help.github.com/articles/about-pull-requests/
+
+## Building Dagger
+
+Dagger is built with [`bazel`](https://bazel.build).
+
+### Setting up Bazel
+
+* Go to [http://bazel.build](http://bazel.build), click on `GET BAZEL`, and install Bazel.
+* Make sure you use Android Studio 3.2 or greater.
+* In *Android Studio*, go to `Preferences > Plugins`.
+  * search for `bazel` and install the plugin.
+  * If no result shows up, click on `Search in repositories`, search for `bazel` and install the plugin.
+
+### Building Dagger from the command line
+
+* Clone the Dagger repository:
+
+```
+git clone git@github.com:google/dagger.git dagger
+```
+
+*  Build the Dagger project:
+
+```
+bazel build
+```
+
+* You can install the Dagger libraries in your local maven repository with the `install-local-snapshot` script.
+* It will build the libraries and install them with a `LOCAL-SNAPSHOT` version.
+* If you get the following error: `ERROR: missing input file '@androidsdk//:build-tools/26.0.2/aapt'`, install the missing build tools version with sdkmanager.
+
+```
+./util/install-local-snapshot.sh
```

Comment:
What's this doing here?

Reply:
The text above was meant to describe what we're doing high level, while this was the actual command. I'll inline.